styleguide/source/_patterns/01-molecules/09-search-results/search-results-columns-external.twig

```diff
@@ -0,0 +1,23 @@
+{#
+ # @file search-results-columns-external.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<div class="layout layout_two_up-large">
+  <div class="layout_two_up-large-primary search-result">
+    <div class="search-result_columns">
```

Comment:
Current functionality uses the js-click-area to do block-level linking (see https://www.ama-assn.org/search/ama-assn/Medicare%20Payment%20Changes%20(MACRA)) but we should stay consistent with the new way of things, which has been block level < a > tags. @deadarm please refactor accordingly.